recirq/quantum_chess/quantum_board.py

```diff
@@ -133,6 +135,10 @@ def undo_last_move(self) -> bool:
                 return False
         return True
 
+    def record_time(self, t0, t1) -> None:
```

Comment:
You could have t1 be an optional argument and if None, set it to time.perf_counter()

Reply:
Thanks Doug for looking into this! The function record_time() is updated.